### PR TITLE
docs: review ini-config.adoc (smoe:docs29_ini_config)

### PR DESCRIPTION
**Pull request by smoe**
This hopefully looks as good as I hope it to look.

### CODE REVIEW

File: docs/src/config/ini-config.adoc [hunk 1]
```diff
@@ -121,23 +116,18 @@ ini.2.max_velocity
...
-Most sample configurations use custom sections and variables to put all of the
-settings into one location for convenience.
+Most sample configurations use custom sections and variables to put all of the settings into one location for convenience.
 
-To add a custom variable to an existing LinuxCNC section, simply include
-the variable in that section.
+To add a custom variable to an existing LinuxCNC section, simply include the variable in that section.
 
-.Custom Variable Example
+.Custom Variable Example, assigning the value 'LINEAR' to the variable 'TYPE', and the value 16000 to the variable 'SCALE'.
```
Comment (hansu) [inline docs/src/config/ini-config.adoc:130]:
```suggestion
.Custom Variable Example, assigning the value 'LINEAR' to the variable 'TYPE', and the value '16000' to the variable 'SCALE'.
```

File: docs/src/config/ini-config.adoc [hunk 2]
```diff
@@ -632,99 +558,88 @@ FEATURES & 0x20 -> OWORD_WARNONLY
 ====
 
 [NOTE]
-[WIZARD]WIZARD_ROOT is a valid search path but the Wizard has not been fully
-implemented and the results of using it are unpredictable.
+`[WIZARD]WIZARD_ROOT` is a valid search path but the Wizard has not been fully implemented and the results of using it are unpredictable.
 
-* 'LOG_LEVEL = 0' Default 0
+* `LOG_LEVEL = 0` Default 0 +
   Specify the log_level (default: -1)
```
Comment (hansu) [inline docs/src/config/ini-config.adoc:564]:
```suggestion
  Specify the log_level (default: 0)
```

File: docs/src/config/ini-config.adoc [hunk 3]
```diff
@@ -632,99 +558,88 @@ FEATURES & 0x20 -> OWORD_WARNONLY
 ====
 
 [NOTE]
-[WIZARD]WIZARD_ROOT is a valid search path but the Wizard has not been fully
-implemented and the results of using it are unpredictable.
+`[WIZARD]WIZARD_ROOT` is a valid search path but the Wizard has not been fully implemented and the results of using it are unpredictable.
 
-* 'LOG_LEVEL = 0' Default 0
+* `LOG_LEVEL = 0` Default 0 +
```
Comment (hansu) [inline docs/src/config/ini-config.adoc:563]:
```suggestion
* `LOG_LEVEL = 0` 
```

File: docs/src/config/ini-config.adoc [hunk 4]
```diff
@@ -632,99 +558,88 @@ FEATURES & 0x20 -> OWORD_WARNONLY
...
 --
-* 'HALFILE = texample.tcl [arg1 [arg2] ...]' - Execute the tcl file 'texample.tcl'
-  at start up with arg1, arg2, etc as argv list.  Files with a .tcl suffix are
-  processed as above but use haltcl for processing  See the
-  <<cha:haltcl,HALTCL Chapter>> for more information.
-* 'HALFILE = LIB:sys_example.hal' - Execute the system library file 'sys_example.hal' at start up.
+* `HALFILE =` _texample.tcl_ [_arg1_ [_arg2_] ...] - Execute the tcl file 'texample.tcl' at start up with arg1, arg2, etc. as argv list.
+  Files with a .tcl suffix are processed as above but use haltcl for processing.
+  See the <<cha:haltcl,HALTCL Chapter>> for more information.
+* ``HALFILE = LIB:``_sys_example.hal_ - Execute the system library file _sys_example.hal_ at start up.
```
Comment (hansu) [inline docs/src/config/ini-config.adoc:629]: This doesn't render correctly

File: docs/src/config/ini-config.adoc [hunk 5]
```diff
@@ -632,99 +558,88 @@ FEATURES & 0x20 -> OWORD_WARNONLY
...
   Explicit use of the LIB: prefix causes use of the system library HALFILE without searching the INI file directory.
-* 'HALFILE = LIB:sys_texample.tcl [arg1 [arg2 ...]]' - Execute the system library file 'sys_texample.tcl' at start up.
+* ``HALFILE = LIB:``_sys_texample.tcl_ [_arg1_ [_arg2_ ...]] - Execute the system library file _sys_texample.tcl_ at start up.
```
Comment (hansu) [inline docs/src/config/ini-config.adoc:631]: This doesn't render correctly

File: docs/src/config/ini-config.adoc [hunk 6]
```diff
@@ -736,68 +651,60 @@ The LIB:halcheck.tcl line should be the last [HAL]HALFILE.
...
 LinuxCNC can start other applications before the specified GUI is started.
-The applications can be started after a specified delay to allow for
-GUI-dependent actions (like creating GUI-specific HAL pins).
+The applications can be started after a specified delay to allow for GUI-dependent actions (like creating GUI-specific HAL pins).
 
-* 'DELAY = value' - seconds to wait beore starting other applications.
-  A delay may be needed if an application has dependencies on [HAL]POSTGUI_HALFILE actions or GUI-created HAL pins
-  (default DELAY=0).
-* 'APP = appname [arg1 [arg2 ...]]' - Application to be started.
+* `DELAY =` _value_ - seconds to wait beore starting other applications.
```
Comment (hansu) [inline docs/src/config/ini-config.adoc:692]:
```suggestion
* `DELAY =` _value_ - seconds to wait before starting other applications.
```

File: docs/src/config/ini-config.adoc [hunk 7]
```diff
@@ -1296,39 +1132,34 @@ latexmath:[ \frac{X\, inches}{1\, encoder\, count} =
...
 //latexmath:[$\frac{volt}{mu\, s}$].
-* 'D = 0' - The derivative gain for the joint servo.
-  The value multiplies the difference between the current and previous errors, resulting in a
-  contribution to the computed voltage for the motor amplifier.
+* `D = 0` - The derivative gain for the joint servo.
+  The value multiplies the difference between the current and previous errors, resulting in a contribution to the computed voltage for the motor amplifier.
   The units on the D gain are volts per machine unit per second, e.g., image:images/i-term.png[height=25]
 // latexmath:[$\frac{volt}{mu/s}$].
-* 'FF0 = 0' - The 0th order feed forward gain.
+* `FF0 = 0` - The 0th order feed forward gain.
```
Comment (hansu) [inline docs/src/config/ini-config.adoc:1153]: after this some ``` ` ` ``` changes are missing until STEP_SCALE

**Review by hansu — CHANGES_REQUESTED** [review]
:+1:

File: docs/src/config/ini-config.adoc [hunk 8]
```diff
@@ -632,99 +558,88 @@ FEATURES & 0x20 -> OWORD_WARNONLY
...
 --
-* 'HALFILE = texample.tcl [arg1 [arg2] ...]' - Execute the tcl file 'texample.tcl'
-  at start up with arg1, arg2, etc as argv list.  Files with a .tcl suffix are
-  processed as above but use haltcl for processing  See the
-  <<cha:haltcl,HALTCL Chapter>> for more information.
-* 'HALFILE = LIB:sys_example.hal' - Execute the system library file 'sys_example.hal' at start up.
+* `HALFILE = texample.tcl` [_arg1_ [_arg2_] ...] - Execute the tcl file 'texample.tcl' at start up with arg1, arg2, etc. as argv list.
+  Files with a .tcl suffix are processed as above but use haltcl for processing.
+  See the <<cha:haltcl,HALTCL Chapter>> for more information.
+* ``HALFILE = LIB:sys_example.hal` - Execute the system library file _sys_example.hal_ at start up.
```
Comment (hansu) [inline docs/src/config/ini-config.adoc:629]:
```suggestion
* `HALFILE = LIB:sys_example.hal` - Execute the system library file _sys_example.hal_ at start up.
```

File: docs/src/config/ini-config.adoc (hunk 7 above)
Comment (hansu) [reply docs/src/config/ini-config.adoc]: I found another one "OFFSET_AV_RATIO" :D

**Comment by smoe** [comment]
I am somewhat afraid of all the changes this drags in with the existing translations.

**Comment by smoe** [comment]
@hansu The error is just a `"E: Failed to fetch http://deb.debian.org/debian/pool/main/y/yapps2/yapps2_2.2.1-3.1_all.deb  Unable to connect to deb.debian.org:http:" - I suggest to ignore this.`

File: docs/src/config/ini-config.adoc (hunk 7 above)
Comment (smoe) [reply docs/src/config/ini-config.adoc]: Done

File: docs/src/config/ini-config.adoc (hunk 4 above)
Comment (smoe) [reply docs/src/config/ini-config.adoc]: Simplified.

**Pull request merged**